.github/workflows/hmis_qa1_ci_cd.yml

```
@@ -0,0 +1,143 @@
+on:
+  push:
+    branches:
+      - hmis-qa1
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)